sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ServiceTierOptions.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.ai.openai.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * Specifies the latency tier to use for processing the request.
+ * This parameter is relevant for customers subscribed to the scale tier service:
+ *  - If set to 'auto', and the Project is Scale tier enabled, the system will utilize scale tier credits until they are exhausted.
+ *  - If set to 'auto', and the Project is not Scale tier enabled, the request will be processed using the default service tier with a lower uptime SLA and no latency guaranty.
+ *  - If set to 'default', the request will be processed using the default service tier with a lower uptime SLA and no latency guaranty.
+ *  - When not set, the default behavior is 'auto'.
+ * When this parameter is set, the response body will include the `service_tier` utilized.
+ */
+public final class ServiceTierOptions extends ExpandableStringEnum<ServiceTierOptions> {
```

Comment:
Any reason why we want to introduce `ServiceTier` and `ServiceTierOptions`? We can keep only ServiceTier and share it in both of the Request/Response classes, ChatCompletions and ChatCompletionsOptions. respectively.

Reply:
The enum values are different in the response model than in the request one, so I wanted to prevent from unnecessarily coupling concepts that may not overlap in the backend. Should there be a change to turn one or the other into an `object` or something along those lines.

Long story short: I am being (maybe) overly careful 😅